Makefile

```diff
@@ -100,10 +100,9 @@ endif
 	./hack/bin/cabal-install-artefacts.sh $(package)
 
 # ci here doesn't refer to continuous integration, but to cabal-run-integration.sh
-# Usage: make ci                        - build & run all tests, excluding integration
-#        make ci package=all            - build & run all tests, including integration
-#        make ci package=brig           - build brig & run "brig-integration"
-#        make ci package=integration    - build & run "integration"
+# Usage: make ci-fast                   - run all integration tests
+#        make ci-fast package=brig      - run integration test for package
```

Comment:
why did you remove `package=all` (genuinely interesting) and `package=integration` (kind of a special case because the ideal is that this is the only integration test suite we have (everything else is legacy)?